pyscf/gto/basis/soecp/ECPDS60MDFSO.dat

```diff
@@ -0,0 +1,208 @@
+ECP
+# Gold atom
+# Q=19., MEFIT, MCDHF+Breit, Ref 37.
+# [37] D. Figgen, G. Rauhut, M. Dolg, H. Stoll, Chem. Phys. 311, 227 (2005).
+Au nelec 60
+Au ul
+2	1.000000	0.000000
+Au S
+2	13.523218	426.641867
+2	6.264384	36.800668
+Au P
+2	11.413867	87.002091	-5.234337
+2	10.329215	174.004370	-6.738142
+Au D
+2	7.430963	49.883655	-174.004182
+2	8.321990	74.684549	174.004370
+2	4.609642	6.486227	-17.741219
+2	3.511507	9.546821	17.902438
+Au F
+2	3.084639	8.791640	-49.883655
+2	3.024743	11.658456	49.789700
+Au G
+2	3.978442	-5.234337	-5.861094
+2	4.011491	-6.738142	5.829228
+# Mercury atom
+# Q=20., MEFIT, MCDHF+Breit, Ref 37.
+# [37] D. Figgen, G. Rauhut, M. Dolg, H. Stoll, Chem. Phys. 311, 227 (2005).
+Hg nelec 60
+Hg ul
+2	1.000000	0.000000
+Hg S
+2	12.413071	275.774797
+2	6.897913	49.267898
+Hg P
+2	11.310320	80.506984	-6.338414
+2	10.210773	161.034824	-8.099863
+Hg D
+2	8.407895	51.137256	-161.013967
+2	8.214086	76.707459	161.034824
+2	4.012612	6.561821	-18.166832
+2	3.795398	9.818070	18.367773
+Hg F
+2	3.273106	9.429001	-51.137256
+2	3.208321	12.494856	51.138306
+Hg G
+2	4.485296	-6.338414	-6.286001
+2	4.513200	-8.099863	6.247428
+# Thallium atom
+# Q=21., MEFIT, MCDHF+Breit+QED, Ref 28.
+# [28] B. Metz, M. Schweizer, H. Stoll, M. Dolg, W. Liu, Theor. Chem. Acc. 104, 22 (2000).
+Tl nelec 60
+Tl ul
+2	1.000000	0.000000
+Tl S
+2	12.167805	281.284663
+2	8.294909	62.434251
+Tl P
+2	7.151492	4.633408	-7.494534
+2	5.172865	9.341756	-9.540575
+2	9.891072	72.299253	-7.797992
+2	9.003391	144.558037	-9.259524
+Tl D
+2	7.130218	35.943039	-9.266817
+2	6.926906	53.909593	9.341756
+2	5.417570	10.381939	-144.598506
+2	5.138681	15.583822	144.558037
+Tl F
+2	5.626399	15.825488	-35.943039
+2	5.548952	21.104021	35.939729
+2	2.874946	2.915127	-10.381939
+2	2.821451	3.896903	10.389215
+Tl G
+2	6.679057	-7.494534	-10.550326
+2	6.706835	-9.540575	10.552010
+2	7.209284	-7.797992	-1.943418
+2	7.070964	-9.259524	1.948451
+# Lead atom
+# Q=22., MEFIT, MCDHF+Breit+QED, Ref 29.
+# [29] B. Metz, H. Stoll, M. Dolg, J. Chem. Phys. 113, 2563 (2000).
+Pb nelec 60
+Pb ul
+2	1.000000	0.000000
+Pb S
+2	12.296303	281.285499
+2	8.632634	62.520217
+Pb P
+2	10.241790	72.276897	-9.096665
+2	8.924176	144.591083	-11.531996
+Pb D
+2	7.754336	35.848507	-144.553795
+2	7.720281	53.724342	144.591083
+2	4.970264	10.115256	-9.517385
+2	4.563789	14.833731	9.940621
+Pb F
+2	3.887512	12.209892	-35.848507
+2	3.811963	16.190291	35.816228
+Pb G
+2	5.691577	-9.096665	-8.139928
+2	5.715567	-11.531996	8.095145
+# Bismuth atom
+# Q=23., MEFIT, MCDHF+Breit+QED, Ref 29.
+# [29] B. Metz, H. Stoll, M. Dolg, J. Chem. Phys. 113, 2563 (2000).
+Bi nelec 60
+Bi ul
+2	1.000000	0.000000
+Bi S
+2	13.043090	283.264227
+2	8.221682	62.471959
+Bi P
+2	10.467777	72.001499	-10.247443
+2	9.118901	144.002277	-12.955710
```

Comment:
Here also should be four P components:
`
Bi P
2	10.467777	72.001499	-10.247443
2	9.118901		144.002277	-12.955710
2	6.754791		5.007945		-10.015890
2	6.252592		9.991550		9.991550
`
www.tc.uni-koeln.de/cgi-bin/pp.pl?language=en,format=molpro,element=Bi,job=getecp,ecp=ECP60MDF